js/twitterFetcher.js

```diff
@@ -285,6 +285,7 @@
             tweet: tweets[n].innerHTML,
             author: authors[n].innerHTML,
             time: times[n].textContent,
+            timestamp: times[n].getAttribute('datetime').replace('+0000', 'Z').replace(/([\+\-])(\d\d)(\d\d)/, '$1$2:$3'),
```

Comment:
I'll break this one into two questions:

1. What is the .replace stuff for?

The .replace calls ensure that the timestamp coming from twitter gets converted into a valid ISO 8601 time string; the time string as provided is not correctly recognized on some older browsers, so passing it into `new Date` doesn't always work.

`+0000` is not supported by all browsers, specifically.  Thus, if it's a GMT time, we just replace it with "Z" which is valid on all browsers and is more common. The second replace will change e.g. -0700 to -05:00 which is, again, more correct to the standard and works on older browsers where some don't support the original format.

These replace calls are needed regardless of what format is returned, a timestamp or a ISO 8601 string.

2. Why use an ISO 8601 time string instead of returning a unix timestamp a la `.getTime()`?

There are two common formats for passing timestamps; unix timestamps and ISO 8601.  Both of those can easily be read by the computer and will work correctly when passed in to `new Date`, but ISO 8601 is human readable and is (in my observation) the most common format used in JSON encoding; in short, we think it's a better format to use. If you really want a unix timestamp we can do that instead, it just seems like this is more generally useful and just as easy to use.